src/css/tabulator.css

```diff
@@ -0,0 +1,1617 @@
+
+	font-size: var(--tabulator-textSize);
+	text-align: left;
+	overflow: hidden;
+
+	transform: translateZ(0);
+
+	&[tabulator-layout="fitDataFill"] {
+		.tabulator-tableholder {
+			.tabulator-table {
+				min-width: 100%;
+			}
+		}
+	}
+
+	&[tabulator-layout="fitDataTable"] {
+		display: inline-block;
+	}
+
+	&.tabulator-block-select {
+		user-select: none;
+	}
+
+	&.tabulator-ranges {
+		.tabulator-cell:not(.tabulator-editing) {
+			user-select: none;
+		}
+	}
+
+	/* column header containing element */
+	.tabulator-header {
+		position: relative;
+		box-sizing: border-box;
+
+		width: 100%;
+
+		border-bottom: 1px solid var(--tabulator-headerSeparatorColor);
+		background-color: var(--tabulator-headerBackgroundColor);
+		color: var(--tabulator-headerTextColor);
+		font-weight: bold;
+
+		white-space: nowrap;
+		overflow: hidden;
+
+		user-select: none;
+
+		outline: none;
+
+		&.tabulator-header-hidden {
+			display: none;
+		}
+
+		.tabulator-header-contents {
+			position: relative;
+			overflow: hidden;
+
+			.tabulator-headers {
+				display: inline-block;
+			}
+		}
+
+		/* individual column header element */
+		.tabulator-col {
+			display: inline-flex;
+			position: relative;
+			box-sizing: border-box;
+
+			flex-direction: column;
+			justify-content: flex-start;
+
+			border-right: 1px solid var(--tabulator-headerBorderColor);
+			background: var(--tabulator-headerBackgroundColor);
+
+			text-align: left;
+			vertical-align: bottom;
+			overflow: hidden;
+
+			&.tabulator-moving {
+				position: absolute;
+				border: 1px solid var(--tabulator-headerSeparatorColor);
+				background: darken(var(--tabulator-headerBackgroundColor), 10%);
+				pointer-events: none;
+			}
+
+			&.tabulator-range-highlight {
+				background-color: var(--tabulator-rangeHeaderHighlightBackground);
+				color: var(--tabulator-rangeHeaderTextHighlightBackground);
+			}
+
+			&.tabulator-range-selected {
+				background-color: var(--tabulator-rangeHeaderSelectedBackground);
+				color: var(--tabulator-rangeHeaderSelectedTextColor);
+			}
+
+			/* hold content of column header */
+			.tabulator-col-content {
+				box-sizing: border-box;
+				position: relative;
+				padding: 4px;
+
+				/* header menu button */
+				.tabulator-header-popup-button {
+					padding: 0 8px;
+
+					&:hover {
+						cursor: pointer;
+						opacity: 0.6;
+					}
+				}
+
+				/* hold title and sort arrow */
+				.tabulator-col-title-holder {
+					position: relative;
+				}
+
+				/* hold title of column header */
+				.tabulator-col-title {
+					box-sizing: border-box;
+					width: 100%;
+
+					white-space: nowrap;
+					overflow: hidden;
+					text-overflow: ellipsis;
+					vertical-align: bottom;
+
+					&.tabulator-col-title-wrap {
+						white-space: normal;
+						text-overflow: initial;
+					}
+
+					/* element to hold title editor */
+					.tabulator-title-editor {
+						box-sizing: border-box;
+						width: 100%;
+
+						border: 1px solid #999;
```

Comment:
Shouldnt the border come from css var too? Ideally here you do not want to hard code theme & typeography settings